docs/specification.md

```diff
@@ -92,6 +92,8 @@ Required `Safelist` entrypoints:
 * `assertReceivers(list address)`
   * Fails if any address in the list is not whitelisted or is blacklisted.
 
+We currently support 2 versions of transferlist: [Ours](https://github.com/tqtezos/stablecoin/blob/bd3c59cf3d084a88d1695829d6f2015018dc5cbc/haskell/src/Indigo/Contracts/Transferlist.hs) and Michael's [lorentz-contract-transferlist](https://github.com/tqtezos/lorentz-contract-transferlist/tree/476c0429adf74041ed7d26c44e5e8e73f5a54f36) located at TODO
```

Comment:
We support any version of Transferlist, as long as its has three entrypoints mentioned above. We can mention TZIP-15 here and say that these entrypoints are from TZIP-15.